src/toast/examples.js

```diff
@@ -0,0 +1,207 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import * as React from 'react';
+import {styled} from '../styles';
+import {Toast, KIND} from './index';
+import {Button, KIND as ButtonKind, SIZE} from '../button';
+
+export const suite = 'Toast Test Suite';
+export const tests = {
```

Comment:
can we move these out to an `example-list.js` file, so the Toast component is ready for e2e tests? We did the same for modal / cards.

Reply:
Sure, I'll add a follow up PR for it